source/dubregistry/registry.d

```diff
@@ -275,6 +304,64 @@ class DubRegistry {
 		return ret;
 	}
 
+	PackageInfo[string] getPackageInfosRecursive(string[] packnames, PackageInfoFlags flags)
```

Comment:
https://github.com/dlang/dub-registry/commit/ca5d2670533d2055fc784157683af01efafaa288#diff-31fe65cc39b31f74454f75b09e88a761 has been added in the meantime and to avoid name conflicts I renamed this method.
It might make sense to unify these, but this one here returns an eager array whereas the other returns a lazy range.